styles/styles.css

```diff
@@ -241,3 +243,24 @@ main .section {
 main .section.highlight {
   background-color: var(--highlight-background-color);
 }
+
+#coming-back-summer-of-2022{
+font-size: 70px;
+}
+
+main > .section > .default-content-wrapper > h1 {
+  font-weight: normal;
+}
+
+#you-want-your-own-supernatural-impala-but-cant-afford-100000-to-buy-one--cant-find-a-1967-chevrolet-impala-even-if-you-could-afford-one--come-share-in-my-love-for-baby-and-take-advantage-of-the-years-of-work-put-into-this-beautiful-car--have-a-private-photo-shoot-be-driven-to-a-wedding-cosplay-party-prom-or-other-special-event--be-the-envy-of-all-your-supernatural-friends-and-create-a-special-event-special-memories-that-will-last-forever, #email-babysupernaturalimpalacom, #text-385-214-5306-and-talk-to-mike-for-more-info, #contact-us{
```

Comment:
What the hell is this?

Reply:
Yes , I was just testing that. I did change them with proper CSS notations, though - 

main > .section > .default-content-wrapper > h1
main > .section > .default-content-wrapper > h4
main > .section > .default-content-wrapper > h3